From 7784b62bd2b64aabd6bfef9ef39ac4f3e8c55c77 Mon Sep 17 00:00:00 2001
From: Alistair Leslie-Hughes <leslie_alistair@hotmail.com>
Date: Tue, 8 Jun 2021 08:56:40 +1000
Subject: [PATCH] bcrypt: Stop compile error when HAVE_GNUTLS_CIPHER_INIT not
 defined
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

In file included from dlls/bcrypt/gnutls.c:1916:
include/windef.h:112:24: error: unknown type name ‘va_list’
  112 | #  define __ms_va_list va_list
      |                        ^~~~~~~
include/winbase.h:2076:84: note: in expansion of macro ‘__ms_va_list’
 2076 | WINBASEAPI DWORD       WINAPI FormatMessageA(DWORD,LPCVOID,DWORD,DWORD,LPSTR,DWORD,__ms_va_list*);

Signed-off-by: Alistair Leslie-Hughes <leslie_alistair@hotmail.com>
---
 dlls/bcrypt/gnutls.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/dlls/bcrypt/gnutls.c b/dlls/bcrypt/gnutls.c
index 37a80bdc062..64825f5b99b 100644
--- a/dlls/bcrypt/gnutls.c
+++ b/dlls/bcrypt/gnutls.c
@@ -25,9 +25,10 @@
 #include "config.h"
 #include "wine/port.h"
 
+#include <stdarg.h>
+
 #ifdef HAVE_GNUTLS_CIPHER_INIT
 
-#include <stdarg.h>
 #include <assert.h>
 #include <gnutls/gnutls.h>
 #include <gnutls/crypto.h>
-- 
2.30.2

